stable/redis/README.md

```diff
@@ -43,39 +49,89 @@ The command removes all the Kubernetes components associated with the chart and
+| `master.livenessProbe.enabled`                    | Turn on and off liveness probe (redis master pod)                                                              | `true`                               |               |
+| `master.livenessProbe.initialDelaySeconds`        | Delay before liveness probe is initiated (redis master pod)                                                    | `30`                                 |               |
+| `master.livenessProbe.periodSeconds`              | How often to perform the probe (redis master pod)                                                              | `30`                                 |               |
+| `master.livenessProbe.timeoutSeconds`             | When the probe times out (redis master pod)                                                                    | `5`                                  |               |
+| `master.livenessProbe.successThreshold`           | Minimum consecutive successes for the probe to be considered successful after having failed (redis master pod) | `1`                                  |               |
+| `master.livenessProbe.failureThreshold`           | Minimum consecutive failures for the probe to be considered failed after having succeeded.                     | `5`                                  |               |
+| `master.readinessProbe.enabled`                   | Turn on and off readiness probe (redis master pod)                                                             | `true`                               |               |
+| `master.readinessProbe.initialDelaySeconds`       | Delay before readiness probe is initiated (redis master pod)                                                   | `5`                                  |               |
+| `master.readinessProbe.periodSeconds`             | How often to perform the probe (redis master pod)                                                              | `10`                                 |               |
+| `master.readinessProbe.timeoutSeconds`            | When the probe times out (redis master pod)                                                                    | `1`                                  |               |
+| `master.readinessProbe.successThreshold`          | Minimum consecutive successes for the probe to be considered successful after having failed (redis master pod) | `1`                                  |               |
+| `master.readinessProbe.failureThreshold`          | Minimum consecutive failures for the probe to be considered failed after having succeeded.                     | `5`                                  |               |
+| `slave.serviceType`                        | Kubernetes Service type (redis slave)                                                                          | `LoadBalancer`                       |               |
```

Comment:
I think the service type for both master and slave svc's should be `ClusterIP` by default